COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
#11557 is the followup. IF that one gets merged I will remove the COPYING change from this one. Post to nix-dev to follow regarding the #11557 to follow later today.